about.html

```diff
@@ -0,0 +1,132 @@
+            <img src="./images/square-twitter.svg" alt="" />
+          </li>
+
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+        <div class="nav-list">
+          <div class="nav-logo">
+            <a href="./index.html"
+              ><h3>AD</h3>
+              <img src="./images/logo.jpg" alt="logo" />
+            </a>
+          </div>
+          <ul class="nav-group">
+            <li class="nav-item menu"><a href="./index.html"> Home</a></li>
+            <li class="nav-item"><a href="#"> About</a></li>
+            <li class="nav-item"><a href="#">Program</a></li>
+            <li class="nav-item"><a href="#">Join</a></li>
+            <li class="nav-item"><a href="#">Sponsor</a></li>
+            <li class="nav-item"><a href="#">News</a></li>
+            <li class="nav-item"><a href="#">Connect</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <section class="home-page">
+      <div class="home-container home-container-1">
+        <h1 class="home-title home-title-1">
+          Design your dream wedding with Andy decor
+        </h1>
+        <p class="home-description home-desc">
+          Our design process starts with getting to know you. How can we design
+          something that reflects you or you’ll like if we don’t know you? From
+          there we create different design ideas based on the initial interview
+          and collaboratively finalise the design.
+        </p>
+        <p class="home-info info">
+          Please contact us per Email for any further questions about Andy
+          decor.
+        </p>
+        <p class="info-detail detail">Andy2015@decor.org</p>
+      </div>
+    </section>
+    <section class="about-page about">
+      <div class="about-container">
+        <h2 class="about-title about-title-1">
+          Designing your dream wedding Logo
+        </h2>
+        <hr />
+        <p class="about-para">
+          The logo of Andy decor was decided through the logo competition from
+          8. June to 7. July.
+        </p>
+        <div class="logo-wraper">
+          <img src="./images/logo.jpg" alt="logo" />
+          <h3 class="logo-name">Andy decor</h3>
+        </div>
+      </div>
+    </section>
+    <section class="about-page about about-1">
+      <div class="about-container">
+        <hr class="separator" />
+        <h2 class="about-title about-title-2">See the past Andy decor</h2>
+        <hr />
+        <p class="about-para">
+          Take a look at the last two Andy decor which took place in Ethiopia
+          and in America.
+        </p>
+        <div class="work-container">
+          <div class="ethiopia">
+            <h4>2013</h4>
+            <p>Andy decor 2013 in Ethiopia</p>
+          </div>
+          <div class="america">
+            <h4>2014</h4>
+            <p>Andy decor 2014 in America</p>
+          </div>
+        </div>
```

Comment:
- [ ] let's use the overlay effect within the past work cards as specified in the [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) design.

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/217754496-0c77bf0b-95bf-4b90-a604-914ceea9c216.png)  |  ![image](https://user-images.githubusercontent.com/60618877/217754059-bea3a194-3941-4891-92e7-ceddd8d60f43.png)